snapcraft/internal/project_loader/_extensions/_utils.py

```diff
@@ -98,12 +98,9 @@ def find_extension(extension_name: str) -> Type[Extension]:
     except ImportError:
         raise errors.ExtensionNotFoundError(extension_name)
 
-    # This may throw an AttributeError, but that would be programmer error of whoever
-    # is hacking on extensions.
-    extension_class_name = "{}Extension".format(
-        extension_name.capitalize().replace("-", "_")
-    )
-    return getattr(extension_module, extension_class_name)
+    # Extensions have an attribute with the name of the extension
```

Comment:
what's the 'name of the extension class'?  Is that the hard-coded "ExtensionImpl"?

Reply:
oh, this slipped from the previous implementation I told you about